documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+
+.Procedure
+Execute the following steps for each `Kafka.spec.cruiseControl` configuration in your Kafka cluster.
+
+. Update the `Kafka` resource in an editor.
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl edit kafka _my-cluster_
+----
+
+. If `type: external` logging is configured in `.spec.cruiseControl.logging`:
++
+Replace the `name` of the ConfigMap containing the logging configuration:
++
+[source,yaml,subs="attributes+"]
+----
+logging:
+  type: external
+  name: my-config-map
+----
++
+With the `valueFrom.configMapKeyRef` field and specify both the ConfigMap `name` and the `key` under which the logging is stored:
++
+[source,yaml,subs="attributes+"]
+----
+logging:
+  type: external
+  valueFrom:
+    configMapKeyRef:
+      name: my-config-map
+      key: log4j2.properties
+----
+
+. If the `.spec.cruiseControl.metrics` field is used to enable metrics:
+
+.. Create a new ConfigMap that stores the YAML configuration for the JMX Prometheus exporter under a key. 
+The YAML must match what is currently in the `.spec.cruiseControl.metrics` field.
++
+[source,yaml,subs="attributes+"]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kafka-metrics
+  labels:
+    app: strimzi
+data:
+  cruise-control-metrics-config.yml: |
```

Comment:
General obseravtion: This would be the only section where we use _.yml_ rather than _.yaml_. Maybe change to be consistent?

Reply:
Updated to .yaml throughout the procedures.